macintacos/vspacecode-config.json

```
@@ -552,128 +572,449 @@
             },
             {
-              "command": "markdown.extension.editing.toggleBold",
-              "key": "b",
-              "name": "Toggle bold",
+              "command": "go.remove.tags",
+              "key": "T",
+              "name": "Remove tags from struct fields",
               "type": "command"
             },
             {
-              "command": "markdown.extension.editing.toggleItalic",
+              "command": "go.import.add",
               "key": "i",
-              "name": "Toggle italic",
+              "name": "Add import",
               "type": "command"
             },
             {
-              "command": "markdown.extension.editing.toggleStrikethrough",
-              "key": "s",
-              "name": "Toggle strikethrough",
+              "command": "go.impl.cursor",
+              "key": "I",
+              "name": "Generate interface stubs",
+              "type": "command"
+            }
+          ],
+          "key": "i",
+          "name": "Insert/remove...",
+          "type": "bindings"
+        },
+        {
+          "bindings": [
+            {
+              "command": "go.languageserver.restart",
+              "key": "R",
+              "name": "Restart language server",
               "type": "command"
             },
             {
-              "command": "markdown.extension.editing.toggleCodeSpan",
-              "key": "`",
-              "name": "Toggle inline code",
+              "command": "go.environment.choose",
+              "key": "c",
+              "name": "Choose Go environment",
               "type": "command"
             },
             {
-              "command": "markdown.extension.editing.toggleCodeBlock",
-              "key": "~",
-              "name": "Toggle code block",
+              "command": "go.gopath",
+              "key": "g",
+              "name": "Show current GOPATH",
               "type": "command"
             },
             {
-              "command": "markdown.extension.editing.toggleList",
+              "command": "go.locate.tools",
               "key": "l",
-              "name": "Toggle list",
+              "name": "Locate configured Go tools",
+              "type": "command"
+            },
+            {
+              "command": "go.tools.install",
+              "key": "i",
+              "name": "Install/update tools",
+              "type": "command"
+            }
+          ],
+          "key": "e",
+          "name": "Environment...",
+          "type": "bindings"
+        },
+        {
+          "bindings": [
+            {
+              "command": "editor.action.revealDefinition",
+              "key": "d",
+              "name": "Go to definition",
+              "type": "command"
+            },
+            {
+              "command": "editor.action.peekDefinition",
+              "key": "D",
+              "name": "Peek definition",
+              "type": "command"
+            },
+            {
```

```suggestion
            {
              "command": "editor.action.revealDefinition",
              "key": "g",
              "name": "Go to definition",
              "type": "command"
            },
```

I know this is a duplicate of `d`, but we could add this one, too, in order to preserve muscle memory of spacemacs users.